.postcssrc

```diff
@@ -0,0 +1,8 @@
+{
+	"modules": false,
+	"plugins": {
+		 	"postcss-import": true,
+    	"postcss-url": true,
+			"autoprefixer": true,
+	},
+}
```

Comment:
[EOF](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

https://eslint.org/docs/rules/eol-last